Add an entry for the XTX xt25f256 32MiB NOR Flash (similar to w25q256).

diff --git a/drivers/mtd/spi-nor/spi-nor.c b/drivers/mtd/spi-nor/spi-nor.c
index d74e94a..4ddd3e0 100644
--- a/drivers/mtd/spi-nor/spi-nor.c
+++ b/drivers/mtd/spi-nor/spi-nor.c	
@@ -2488,6 +2488,7 @@ static const struct flash_info spi_nor_ids[] = {
 
 	/* XTX */
 	{ "xt25f128b", INFO(0x0b4018, 0, 64 * 1024, 256, SECT_4K) },
+	{ "xt25f256", INFO(0x0b4019, 0, 64 * 1024, 512, SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
 
 	/* Catalyst / On Semiconductor -- non-JEDEC */
 	{ "cat25c11", CAT25_INFO(  16, 8, 16, 1, SPI_NOR_NO_ERASE | SPI_NOR_NO_FR) },
 
